platform/android/MapboxGLAndroidSDK/src/main/java/org/maplibre/android/annotations/Circle.java

```diff
@@ -1,356 +1,356 @@
-package org.maplibre.android.annotations;
```

Comment:
This file was only added in the previous PR: https://github.com/maplibre/maplibre-native/pull/1552/files#diff-ba97786bd63cfe6fd21979b009f121f02ef13d8038a02dc9c72624d07fddd585

It was never a part of `maplibre-native`'s main branches. Backwards compatibility is given because it will still be possible to use the old plugin. (We might need a new plugin version if any breaking changes happen regarding the APIs that the plugin is using, now to think of it.)